Makefile

```diff
@@ -135,14 +135,14 @@ e2e-local:
 
 .PHONY: run
 run: ## Run the operator outside of a cluster in development mode
-	operator-sdk run --local \
+	operator-sdk run local \
```

Comment:
At a minimum we should use the vendored version of the operator-sdk `./bin/operator-sdk` so that this doesn't (a) require the dev to install their own version, and (b) sometimes break depending on what version that is.